tdrs-backend/tdpservice/parsers/test/test_parse.py

```diff
@@ -984,3 +985,47 @@ def test_parse_ssp_section3_file(ssp_section3_file):
     assert first.NUM_RECIPIENTS == 51355
     assert second.NUM_RECIPIENTS == 51696
     assert third.NUM_RECIPIENTS == 51348
+
+@pytest.fixture
+def tribal_section_1_file(stt_user, stt):
+    """Fixture for ADS.E2J.FTP4.TS06."""
+    return util.create_test_datafile('ADS.E2J.FTP1.TS142', stt_user, stt, "Tribal Active Case Data")
+
+@pytest.mark.django_db()
+def test_parse_tribal_section_1_file(tribal_section_1_file):
+    """Test parsing Tribal TANF Section 1 submission."""
+    parse.parse_datafile(tribal_section_1_file)
+
+    assert Tribal_TANF_T1.objects.all().count() == 121
+    assert Tribal_TANF_T2.objects.all().count() == 199
+    assert Tribal_TANF_T3.objects.all().count() == 213
+
+    t_t1_objs = Tribal_TANF_T1.objects.all().order_by("CASH_AMOUNT")
+    t_t2_objs = Tribal_TANF_T2.objects.all().order_by("MONTHS_FED_TIME_LIMIT")
+    t_t3_objs = Tribal_TANF_T3.objects.all().order_by("EDUCATION_LEVEL")
```

Comment:
i noticed that the case aggregates columns in the frontend don't populate for tribal submissions
![image](https://github.com/raft-tech/TANF-app/assets/111305129/367397a0-36cb-4416-a2b0-72dabf92fa4a)

i double checked ssp and tanf section 1 and both work. it's not immediately clear to me why tribal does not, it could be an unfinished implementation of the `get_prog_from_section` util